config/ec2/config.go

```diff
@@ -237,6 +242,9 @@ func Configure(p *config.Provider) {
 		r.References["nat_gateway_id"] = config.Reference{
 			Type: "NATGateway",
 		}
+		r.References["destination_prefix_list_id"] = config.Reference{
+			Type: "ManagedPrefixList",
```

Comment:
Please see the comment above:
```suggestion
			TerraformName: "aws_ec2_managed_prefix_list",
```